packages/react-components/src/components/ScreenShareButton.tsx

```diff
@@ -80,22 +80,26 @@ export const ScreenShareButton = (props: ScreenShareButtonProps): JSX.Element =>
 
 const screenshareButtonStyles = (theme: Theme): IButtonStyles => ({
   rootChecked: {
-    background: theme.palette.themePrimary,
-    color: DefaultPalette.white,
+    background: theme.semanticColors.primaryButtonBackground,
+    color: theme.semanticColors.primaryButtonText,
     ':focus::after': { outlineColor: `${DefaultPalette.white} !important` }, // added !important to avoid override by FluentUI button styles
```

Comment:
Should we also update the :focus::after outline color here and on hover below?

Reply:
They're actually intentional - the black outline is really hard to see, I'd only want to update if we get A11y complaining about it